rmgpy/species.py

```diff
@@ -50,6 +50,7 @@
 
 import rmgpy.quantity as quantity
 
+from rmgpy.molecule.molecule import Atom, Bond, Molecule
```

Comment:
There's more discussion in #1198. The only reason this hasn't broken RMG is because the species module is cythonized and the `.pxd` file still has this import. Having it in the `.py` file is good for code clarity, and necessary if we try to run in pure python mode (not that it's working right now).